src/ten.jl

```diff
@@ -11,15 +11,22 @@ ten(d::Real, m::Real=0, s::Real=0) =
 # TODO: improve performance, if possible.  There are a couple of slow tests to
 # make parsing of the string work.
 function ten(strng::AbstractString)
```

Comment:
I assume you wanted to change the name of the argument to
```suggestion
function ten(coordinate::AbstractString)
```
? :slightly_smiling_face:

Reply:
Oops! You're right. Silly oversight.